example/generator/generator_run.sh

```diff
@@ -6,6 +6,12 @@
 # SPDX-License-Identifier:     BSD-3-Clause
 #
 
+if [ -n "${ODP_PLATFORM}" -a "x${ODP_PLATFORM}" != "xlinux-generic" ]
+then
+	echo "Skipping test on probably unsupported platform"
```

Comment:
I thought about doing like that but decided that message might be confusing for people using odp first time. They can think that generator is completely not for this platform.

Reply:
Should we rename then to `generator_null_test.sh` ?